etc/config/c++.amazon.properties

```diff
@@ -634,3 +634,9 @@ libs.vcl.versions=130
 libs.vcl.url=https://www.agner.org/optimize/#vectorclass
 libs.vcl.versions.130.version=1.30
 libs.vcl.versions.130.path=/opt/compiler-explorer/libs/vcl/v1.30
+libs.outcome.versions=trunk
+libs.outcome.url=https://github.com/ned14/outcome.git
```

Comment:
We usually reserve this for things like library homepages / documentations, not for the git repostory itself. Could you change it to, perhaps, the GitHub repostory instead, if it lacks a proper homepage? Thanks! :)